.gitignore

```diff
@@ -21,9 +21,9 @@ yarn-error.log*
 /bundle
 /bundle0
 /package-lock.json
+/pnpm-lock.yaml
```

Comment:
We should always keep the lock file in git, see:
https://pnpm.io/git#:~:text=You%20should%20always%20commit%20the,able%20to%20skip%20package%20resolution

Reply:
Oh I though since we were ignoring lock.json we should have ignored lock.yaml I will revert it